toolchain-m68k

```diff
@@ -241,6 +241,14 @@ def build():
                               path.join('{host}', 'bin'),
                               environ['PATH']])
 
+  """
+  When we have a working compiler in our path, we can also check if the
+  required headers/libraries are present.
+  """
+
+  require_header('<python2.7/python.h>')
```

Comment:
Hmm... I don' think I use C-to-Python bindings anywhere. Isn't it better to just check Python [version](https://docs.python.org/2/library/sys.html#sys.version_info) ?